soul-plugin/soul-plugin-sign/src/test/java/org/dromara/soul/plugin/sign/service/DefaultSignServiceTest.java

```diff
@@ -0,0 +1,201 @@
+import org.dromara.soul.plugin.sign.cache.SignAuthDataCache;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mock.http.server.reactive.MockServerHttpRequest;
+import org.springframework.mock.web.server.MockServerWebExchange;
+import org.springframework.web.server.ServerWebExchange;
+
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ * DefaultSignService Test.
+ *
+ * @author Phoenix Luo
+ * @version 2020/12/5
+ **/
+@RunWith(MockitoJUnitRunner.class)
+@Slf4j
+public final class DefaultSignServiceTest {
+    
+    private SignService signService;
+    
+    private ServerWebExchange exchange;
+    
+    private final String appKey = "D1DFC83F3BC64FABB89DFBD54E5A28C8";
+    
+    private final String secretKey = "692C479F98C841FCBEB444B7CA775F63";
+    
+    private SoulContext passed;
+    
+    @Value("${soul.sign.delay:5}")
+    private int delay;
+    
+    @Before
+    public void setup() {
+        this.signService = new DefaultSignService();
+        this.exchange = MockServerWebExchange.from(MockServerHttpRequest.get("localhost").build());
+        
+        final String path = "/test-api/demo/test";
+        PluginData signData = new PluginData();
+        signData.setId("1");
+        signData.setName(PluginEnum.SIGN.getName());
+        signData.setEnabled(true);
+        signData.setRole(1);
+        BaseDataCache.getInstance().cachePluginData(signData);
+        
+        AppAuthData authData = new AppAuthData();
+        authData.setAppKey(appKey);
+        authData.setAppSecret(secretKey);
+        authData.setEnabled(true);
+        AuthPathData authPathData = new AuthPathData();
+        authPathData.setAppName("test-api");
+        authPathData.setPath(path);
+        authPathData.setEnabled(true);
+        authData.setPathDataList(Lists.newArrayList(authPathData));
+        SignAuthDataCache.getInstance().cacheAuthData(authData);
+        
+        this.passed = new SoulContext();
+        this.passed.setModule("/test-api");
+        this.passed.setMethod("/demo/test");
+        this.passed.setRpcType("springCloud");
+        this.passed.setHttpMethod("GET");
+        this.passed.setPath(path);
+        final String timestamp = String.valueOf(System.currentTimeMillis());
+        this.passed.setTimestamp(timestamp);
+        this.passed.setSign(buildSign(secretKey, timestamp, this.passed.getPath()));
+        this.passed.setAppKey(appKey);
+        this.passed.setContextPath("/test-api");
+        this.passed.setRealUrl("/demo/test");
+    }
+    
+    @Test
+    public void normalTest() {
+        this.exchange.getAttributes().put(Constants.CONTEXT, this.passed);
+        
+        Pair<Boolean, String> ret = this.signService.signVerify(this.exchange);
+        assert ret.getLeft() : "sign正向测试未通过！";
```

Comment:
Why don't you use assert from junit. Please keep consistent with other test classes.

Reply:
Sorry I will fix this